### ISSUE TITLE
the loss is nan

### ISSUE THREAD

**Issue by lostlose** [yjh0410/yolov2-yolov3_PyTorch#53]
Hi, I met the same problem in https://github.com/yjh0410/PyTorch_YOLOv3/issues/1 when I train yolov3 in this project.

**Comment by yjh0410** [comment]
@lostlose 
Please try my another YOLO project: https://github.com/yjh0410/PyTorch_YOLO-Family

**Comment by lostlose** [comment]
@1023280072 Thank you! Due to memory constraints, I can only set the batch_size to 16, and now I'm trying to adjust the learning rate to get the correct results.

**Comment by lostlose** [comment]
@yjh0410 OK, thanks, I will try it later!

**Comment by aknirala** [comment]
It is nan even at batch size 48:
I was using RN50 as backbone, and trying to train coco dataset.
